HW-1: Реализовано 3 метода

kkaidd opened the pull request:
Реализовано создание животных в классах CreateAnimalService и CreateAnimalServiceImpl"
Реализован интерфейс Animal с методами getBreed(), getName(), getCost(), getCharacter().
Создан абстрактный класс AbstractAnimal, содержащий поля для хранения породы (breed), имени (name), цены (cost) и характера (character) животного.

Commit:
HW-1: Реализовано 3 метода

nickdanil commented on lesson_3_Homework/src/main/java/ru/mts/homework/CreateAnimalServiceImpl.java:
```diff
@@ -0,0 +1,54 @@
+        do {
+            createAnimal();
+            count++;
+        } while (count < 3);
```
оздаем CreateAnimalService c default методом по созданию **10** уникальных животных при помощи метода while

Commit:
HW-1.1:
- Исправил перегруженный метод
- Исправил вывод дефолтного метода, раньше он не выводился (сейчас он выводит 10 животных)
- Стал лучше понимать код (ﾉ◕ヮ◕)ﾉ*:･ﾟ✧
